tests/src/Recipe/RecipeEntity.php

```diff
@@ -94,24 +92,21 @@ public function testSuccessRecipeEntity()
         $expected .= "</div>";
         $this->assertEquals($expected, $result);
     }
-
     /**
-     * Testing for failure in generating HTML. Passed integers as opposed to strings.
+     * Testing for failure in generating HTML. Passed inergers as opposed to strings.
      *
      * @return void
      */
-    public function testFailureRecipeEntityGenerateHTML()
-    {
-        $testIngredient = new GRUB\RecipeEntity (9,9,9,9);
-        $result = $testIngredient->generateHTML();
+    public function testFailureRecipeEntityGenerateHTMLSaved(){
+        $testIngredient = new GRUB\Recipe\RecipeEntity (9,9,9,9);
+        $result = $testIngredient->GenerateHTMLSaved();
         $expected = "<div class='recipe'>";
         $expected .= "<div class='left'>";
         $expected .= "<div class='recipeImage'>";
         $expected .= "<img src='9'/>";
         $expected .= "</div>";
-        $expected .= "<a href='9'>";
-        $expected .= "<button>Link to recipe</button>";
-        $expected .= "</a>";
+        $expected .= "<a name='deleteRecipe'><button>Delete Recipe</button></a>";
```

Comment:
Needs hidden form, see marcs addition to recipeEntitiy

Reply:
Fixed